README.md

```diff
@@ -1,77 +1,76 @@
-# scikit-learn-knn-regression
+# sknnr
 
-This package is in active development.
+> ⚠️ **WARNING: sknnr is in active development!** ⚠️
 
-## Developer Guide
+## What is sknnr?
 
-### Setup
+`sknnr` is a package for running k-nearest neighbor (kNN) imputation methods using estimators that are fully compatible with [`scikit-learn`](https://scikit-learn.org/stable/). Notably, common methods such as most similar neighbor (MSN, Moeur & Stage 1995), gradient nearest neighbor (GNN, Ohmann & Gregory, 2002), and random forest nearest neighbors (RFNN, Crookston & Finley, 2008) are included in this package.
```

Comment:
Do you think we should add a footnote on "imputation" to explain that this is distinct from `sklearn`-style missing data imputation? Or do you think that's already clear in context?

Reply:
That's a really good idea (based on some of our previous conversations and the common usage of "impute" in `sklearn`).  Perhaps something like:
> Here we use the term `imputation` to mean that a target's predicted feature values are derived from its *k*-nearest neighbors and not in the usual `scikit-learn` usage as a pre-filling strategy for missing input data.

Please wordsmith as you would like.

Reply:
Great! I modified it slightly to make sure it was clear if someone didn't notice the footnoted text, and linked to `KNNImputer`.